cli/blkio.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/alibaba/pouch/apis/types"
+	units "github.com/docker/go-units"
+)
+
+// WeightDevice defines weight device
+type WeightDevice struct {
+	values []*types.WeightDevice
+}
+
+func validateWeightDevice(val string) (*types.WeightDevice, error) {
```

Comment:
I do not think it is a good way to use a validate function to return a point of real instance. Usually, we just return error using function `validateWeightDevice `. Then from my own point of view, I suggest that we split validation and construct instance from this function into two separated function. WDYT? @Ace-Tang

Reply:
`split validation and construct instance from this function into two separated function` , don‘t you think it's redundancy, get instace will still go into valid logic. Or we can rename function

Reply:
let's make the function named `getValid*`